myapp/app/controllers/users_controller.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :check_login_status
+  before_action lambda  {
+    check_login_status
+    check_user_role
+  }
```

Comment:
lambdaで指定するより各メソッドをシンボルで指定するほうがシンプルでよく見るスタイルかと思いますがどうでしょうか？

```suggestion
  before_action :check_login_status
  before_action :check_user_role
```

Reply:
今更ながら別行にする必要がないので↓でも

```ruby
before_action :check_login_status, :check_user_role
```

Reply:
承知しました。そうします。